keyboards/custommk/ergostrafer/mcuconf.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2023 customMK
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include_next <mcuconf.h>
+
+// Used for audio (?)
+#undef STM32_PWM_USE_TIM4 //timer 4 channel ?
```

Comment:
Yes. PWMD4 uses Timer 4. Channel isn't specified here, but via the config.
```suggestion
#undef STM32_PWM_USE_TIM4
```

Reply:
Yep, was just a leftover comment from earlier, I thought I had deleted it (whereas now I know I've deleted it)